database/seeders/DatabaseSeeder.php

```diff
@@ -2,16 +2,13 @@
 
 namespace Database\Seeders;
 
-// use Illuminate\Database\Console\Seeds\WithoutModelEvents;
 use Illuminate\Database\Seeder;
 
 class DatabaseSeeder extends Seeder
 {
-    /**
-     * Seed the application's database.
-     */
     public function run(): void
     {
-        $this->call(RoleSeeder::class);
+        $this->call(RolesAndPermissionsSeeder::class);
```

Comment:
```suggestion
        $this->call([
            RolesAndPermissionsSeeder::class,
            UserSeeder::class.
        ]);
```

The `Seeder`'s `call` method accepts an array of seeders, which means instead of having multiple `$this->call()` you can have only one.